ethereum/core/src/main/java/org/hyperledger/besu/ethereum/core/UnformattedDataWrapper.java

```diff
@@ -0,0 +1,66 @@
+/*
+ *
+ * Copyright ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+package org.hyperledger.besu.ethereum.core;
+
+import org.hyperledger.besu.plugin.data.UnformattedData;
+
+import java.util.Objects;
+
+import org.apache.tuweni.bytes.Bytes;
+
+/** Wrapper for a Bytes value to be exposed as UnformattedData. */
+public class UnformattedDataWrapper implements UnformattedData {
```

Comment:
nit; a catchy name for this class could be something like `UnformattedBytes`, as it is `Bytes` data type conforming to the `UnformattedData` interface 😉

Reply:
Probably keep the ugly name to keep this top of mind and fix this better soon.